README.md

```diff
@@ -1 +1,194 @@
-# book-an-appointment
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b> 🏖 Motorbike Booking 🏞 </b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [💸 Motorbike Booking ](#-Motorbike-booking-)
+- [🛠 Built With ](#-built-with-)
+  - [📌 Tech Stack ](#-tech-stack-)
+- [🎲 Key Features ](#-key-features-)
+- [🚀 Live Demo](#-live-demo-)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors ](#-authors-)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing ](#-contributing-)
+- [⭐️ Show your support ](#️-show-your-support-)
+- [🙏 Acknowledgments ](#-acknowledgments-)
+- [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Motorbike Booking <a name="about-project"></a>
+
+**Motorbike Booking** is a web application that offers a diverse selection of luxurious accommodations and amenities. Find your slice of paradise, effortlessly plan your trip, and make memories that last a lifetime with MotorbikeHub.
+
+
+# 🛠 Built With <a name="built-with"></a>
+
+## 📌 Tech Stack <a name="tech-stack"></a>
+
+| Domain | Tech Stack |
+|--------|--------|
+| Server | <a href="https://github.com/facebook/create-react-app">React</a> |
+| Library | <a href="https://redux.js.org/">Redux</a> |
+| Compiler | <a href="https://babeljs.io/docs/">Babel</a> |
+
+<!-- Features -->
+
+# 🎲 Key Features <a name="key-features"></a>
+
+- Create a new account
+- Log in to an existing account
+- Sign out from a session
+- Choose from a variety of beatiful Motorbikes
+- Book a Motorbike 
+- Add a Motorbike
+- Delete a Motorbike 
+
+# 🚀 Live Demo <a name="live-demo"></a>
+
+👉 [Not Available](https://....)
+
+# 💻 Rails API <a name="rails-api"></a>
+
+```sh
+  https://github.com/MenendezON/book-an-appointment
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+# 💻 Getting Started with Create React App <a name="getting-started"></a>
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+Clone this repository to your desired folder:
+
+```sh
```

Comment:
- [ ] OPTIONAL: There is no way in the frontend to logout.

The only 2 ways to logout is by reloading the page or deleting local storage manually.

I highly suggest you include a way to logout, in the form of a link or button (it can be simple).

That way, it will be possible to log in with multiple users. 👍 

_tip: Just delete local storage and redirect to the login page.   Also, since your app loses the session upon reloading the page, you could you that._